packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -200,19 +200,17 @@ export class AtomicCategoryFacet
     return !!this.facetState.parents.length;
   }
 
-  private get numberOfSelectedValues() {
-    return this.facetState.values.filter(({state}) => state === 'selected')
-      .length;
-  }
-
   private renderHeader() {
     return (
       <FacetHeader
         i18n={this.bindings.i18n}
         label={this.label}
-        numberOfSelectedValues={this.numberOfSelectedValues}
+        numberOfSelectedValues={
```

Comment:
I understand why passing just `1` or `0` works for `FacetHeader` when I read the current code.

However this seems a potential bug down the road if we ever need to actually know the real number of selected values in `FacetHeader` for any reason.

Reply:
You cannot select more than 1 value/filter with the category facet, so this value will always be accurate (expect if we refactor the feature completely to allow multiple paths). For the isCollapsed tho it's a bit more risky if the facet header ever changes it's rendering condition, but we have cypress tests covering it, so we will know.